python/ray/data/_internal/planner/plan_read_op.py

```diff
@@ -57,10 +60,15 @@ def get_input_data(target_max_block_size) -> List[RefBundle]:
         assert (
             parallelism is not None
         ), "Read parallelism must be set by the optimizer before execution"
+        start = time.time()
```

Comment:
Thanks! Could we also documents in this PR that:

With X amount a read tasks, What is this two timing changed before & after the PR?